web/src/views/SubmissionPortal/Components/LoginPrompt.vue

```diff
@@ -12,15 +12,17 @@ export default defineComponent({
 
 <template>
   <v-alert
-    class="mb-0"
+    class="mt-6 mb-0"
     text
   >
     <v-container>
       <v-row align="center">
         <v-col class="justify-content-center">
+          <h3 class="text-h6 font-weight-bold mb-3">
+            Login to submit data
+          </h3>
```

Comment:
We shouldn't use the `h3` tag then re-assign the style as `h6` mainly because I don't think the `h3` is necessary here and becomes confusing.

Reply:
outdated in current model